compiler/z/codegen/ConstantDataSnippet.cpp

```diff
@@ -1010,90 +1029,99 @@ TR_Debug::print(TR::FILE *pOutFile, TR::S390ConstantDataSnippet * snippet)
       trfprintf(pOutFile, "DC   \t0x%016lx  # Method index", *(intptrj_t*)bufferPos);
       bufferPos += refSize;
 
-      // zero cache slot
-      if (static_cast<TR::S390InterfaceCallDataSnippet*>(snippet)->getNumInterfaceCallCacheSlots() == 0)
-         {
-         printPrefix(pOutFile, NULL, bufferPos, sizeof(intptrj_t));
-         trfprintf(pOutFile, "DC   \t0x%016lx  # flags", *(intptrj_t*)bufferPos);
-         bufferPos += refSize;
-         }
+      printPrefix(pOutFile, NULL, bufferPos, sizeof(intptrj_t));
+      trfprintf(pOutFile, "DC   \t0x%016lx  # Interpreter thunk", *(intptrj_t*)bufferPos);
+      bufferPos += refSize;
 
-      // non-single cache slots
       bool isSingleDynamicSlot = comp()->getOption(TR_enableInterfaceCallCachingSingleDynamicSlot);
```

Comment:
This variable is only used on the else path of the following block, so there is no need to have the preemptive `comp()->getOption` call outside of the else path.

Reply:
Fixed.